Cargo.toml

```diff
@@ -11,17 +12,15 @@ repository = "https://github.com/emk/compose_yml"
 
 [dependencies]
 clippy = { version = "0.*", optional = true }
-error-chain = "0.5"
-lazy_static = "1.0"
-log = "0.3"
-regex = "0.2"
+error-chain = "0.12"
+lazy_static = "1.4"
+log = "0.4"
+regex = "1.3"
```

Comment:
For dependencies with versions 1 and above, we could specify just a major version. In this case, Cargo can pick up a package with this major version and any minor/patch version. It is because minor version changes shouldn't make incompatible API modification. See [semver.org](https://semver.org).